README.md

```diff
@@ -1 +1,156 @@
-# To-do-list
\ No newline at end of file
+# AWESOME BOOK PROJECT
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [AWESOME BOOK PROJECT] <a name="about-project"></a>
+
+**[AWESOME_BOOK_PROJECT]** 
+This project is about addition and deletion of books into local storage 
+using javascript objects. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Javascript
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **[Display_list_in_Table_form]**
+- **[Add_buttons_and_functionallity]**
+- **[Remove_Button_and_functionallity]**
+- **[ES6_Modules]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- In order to run this project you need:
+   - Modern web browser  
```

Comment:
- [ ] Making your README very descriptive is a professional practice as a developer. Unfortunately, you have not fulfilled that requirement. So I would suggest you properly highlight the steps in getting started with this project to make it easy for anyone to either contribute or review this project.